Source/MonoGame.Extended/FPSCounter.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.Xna.Framework;
+
+namespace MonoGame.Extended
+{
+    public static class FPSCounter
```

Comment:
Why is the class static? What if someone wants to have more than one instance of it?